draft-private-access-tokens.md

```diff
@@ -173,14 +185,83 @@ Mediators advertise an Oblivious HTTP URI template {{!RFC6570}} for proxying
 protocol messages to Issuers. For example, one template
 for the Mediator might be https://mediator.net/relay-access-token-request.
 
-# Issuance
+# Protocol
```

Comment:
Do we want to only have a purely interactive protocol?

While interactivity does improve some of the security properties through immediate redemption binding, it still seems nice to have a non-interactive protocol to avoid high cost latency issues, or even requiring the mediator to have to store state for non-authenticated clients.

Reply:
I'm not sure the flexibility is warranted here, but maybe we could support both.

Reply:
Yeah, agreed that the interactive one is cleaner/simpler. Maybe just have the interactive mode for now, and maybe add a note somewhere that a non-interactive construction is possible in case an infra/use-case warrants it?